neuralmonkey/attention/scaled_dot_product.py

```diff
@@ -154,9 +154,18 @@ def attention(
 
     # For multi-head attention, queries, keys and values are linearly projected
     if num_heads > 1:
-        queries = tf.layers.dense(queries, queries_dim, name="query_proj")
-        keys = tf.layers.dense(keys, queries_dim, name="keys_proj")
-        values = tf.layers.dense(values, queries_dim, name="vals_proj")
+        queries = tf.layers.dense(queries,
```

Comment:
Já bych to radši odsadil a dal na jeden řádek pod `tf.layers.dense(`, tyhle sloupečky mi nepřijdou moc přehledný.